server/common/utils/helpers.js

```diff
@@ -331,6 +333,23 @@ const enumerable = namespace => (...keys) =>
     fromEntries(keys.map(key => [key, [namespace, key].join('/')]))
   );
 
+const forEach = f => o =>
```

Comment:
what are f and o?
Please use words as arguments, not letters as it's not possible to understand what it is